src/index.html

```diff
@@ -3,11 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="body">
+
+    <div class="card" data-qa="card">
+
+      <div class="card__computer">
+        <img class="card__img"
+        src="./images/imac.jpeg"
+        alt="Imac">
+      </div>
+
+      <p class="card__imac">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+      <div class="card__product-code">Product code: 195434</div>
+      <div class="card_star-reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card_reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-name">Price:</p>
+        <p class="card__price-sum">$2,199</p>
+      </div>
+
+      <a
+      class="card_link"
+      href="#bay"
+      data-qa="hover"
```

Comment:
don't forget about indentation
```suggestion
        class="card_link"
        href="#bay"
        data-qa="hover"
```